app/bundles/LeadBundle/Resources/views/FormTheme/Filter/_leadlist_filters_entry_widget.html.twig

```diff
@@ -25,8 +26,12 @@
             </div>
         </div>
         <div class="panel-body">
-            <div class="col-xs-6 col-sm-3 field-name">
-                <span><i class="object-icon {{ class }}" aria-hidden="true"></i> {% if isPrototype %}__label__{% else %}{{ fields[object][filterType]['label'] }}{% endif %}</span>
+            <div class="col-xs-2 col-sm-1 ai-center d-flex">
```

Comment:
@andersonjeccel, please help me style this.
<img width="1117" alt="Screenshot 2024-08-30 at 8 28 33 PM" src="https://github.com/user-attachments/assets/3ceceaeb-357c-4b4f-b2bd-c7e523ee73fb">

Thanks